controllers/admin/AdminStatsTabController.php

```diff
@@ -167,9 +167,12 @@ public function displayMenu()
         return $tpl->fetch();
     }
 
-    public function checkModulesNames($a, $b)
+    public function checkModulesNames($a, $b): int
     {
-        return (bool) ($a['displayName'] > $b['displayName']);
+        if ($a['displayName'] == $b['displayName']) {
```

Comment:
Not the purpose of this PR but I can see only one usage of this method, it's in this controller itself. It should be private

Why don't you we use `strcasecmp` https://www.php.net/manual/en/function.strcasecmp.php

If it's only for sorting, it'll be enough and the `checkModulesNames` method can be deleted

`uasort($modules, 'strcasecmp');`

Reply:
`strcasecmp(): Argument #1 ($string1) must be of type string, array given`

Reply:
You can't do `uasort($modules, 'strcasecmp');` as `$modules` is an array of array, it works only when the content of the array is a string.